tests/ui/associated-types/associated-types-impl-redirect.rs

```diff
@@ -1,7 +1,7 @@
-// run-pass
-#![allow(dead_code)]
-#![allow(unused_mut)]
-#![allow(unused_imports)]
```

Comment:
I assume you used some script or regex to remove the attributes.
Could you tweak it to remove lines entirely, instead of leaving empty lines?